config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -180,8 +180,8 @@ periodics:
     preset-e2e-scalability-periodics: "true"
     preset-e2e-scalability-periodics-master: "true"
   annotations:
-    testgrid-dashboards: sig-scalability-kubemark
-    testgrid-tab-name: kubemark-100-scheduler
+    testgrid-dashboards: sig-scalability-scheduler
```

Comment:
It seems the amount of scheduler jobs is increasing over time. Can we extract them to a separate file, e.g.
sig-scalability-scheduler-periodic-jobs.yaml
or sth like that?

Reply:
Also - please move these changes to a separate PR.

Reply:
Done, removed the dashboard change from this PR, I'll open a separate PR for that + a new jobs YAML file.